config/locales/en/casework.yml

```diff
@@ -65,6 +65,7 @@ en:
     codefendants: Co-defendants
     correspondence_address: Correspondence address
     criminal_applications_team: CAT 1
+    criminal_applications_team_2: CAT 2
```

Comment:
Does the national crime team locale need removing too?

Reply:
I've left it in as it could be in some test event data on staging. can remove later.

Reply:
Good point!